tests/src/r2rdump/BasicTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Xunit;
+using R2RDump;
+using System.Xml;
+using System.IO;
+using System.Xml.Linq;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit.Abstractions;
+
+namespace R2RDump.Test
+{
+    public class BasicTest
+    {
+        private readonly ITestOutputHelper _output;
+        public BasicTest(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        [Fact]
+        public void NoSections()
+        {
+            R2RReader r2r = new R2RReader("c:/Code/r2rdump/peFiles/NoSections.ni.dll");
```

Comment:
Please don't hardcode any path in the test. This will work on your machine only.

Reply:
I will change this. I'm still trying to figure out how to pass the path to the test